docs/server-side/v2/api-reference/room-codes/room-codes-overview.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Overview
+nav: 4.1
+---
+
+Room codes are unique encrypted short codes that carry information such as room_id, role and room code state. A Room Code is always associated to specific to a role and room_id combination.
+
+
+## Why use Room Codes?
+
+
+<Note>
+**Room Links with Room Codes**
+
+Construct Prebuilt room links or your custom domain room links format; embed and share with your users
+
+###### How to use room codes to construct room links?
+
+1. **[Prebuilt Room Links→](/concepts/v2/concepts/Prebuilt)**
+2. **[Custom Room Links→](/concepts/v2/concepts/customUI)**
+
+</Note>
+
+<Note>
+**Room Code Authentication**
+
+Power role specific room code authentication required to join a Room when using 100ms SDKs 
+
+###### How to join a room using a Room Code?
+
+
+1. Prebuilt has built-in support for Room Code based authentication
+2. **Use Room Codes to join a room using SDKs →**
+</Note>
+
+Essentially, Room Codes allow you to create unique room links and room code authentication mechanism for every role in your room. For instance, consider three roles created in a Template with their unique room codes.
+
+| Role   | Room Codes     | 
+| :----- | :------- | 
+| Broadcaster | `wms-xbx-qpx`|  
+| Viewer | `uiw-rwr-weo` |  
+| Moderator | `erw-gdo-wej` |
+
+Room Links for each role
+- 100ms Prebuilt link : `https://<template-subdomain>/meeting/<room-code>`
+- Your custom domain link : `https://<your-domain>/meeting/<room-code>`
+
+
+## Room Codes Basics
```

Comment:
i think it would be useful to use markdown headings (h3 - instead of bold text) to represent the titles in this section. current preview looks like a blob of unreadable text - headings will help

Reply:
this is done.